url.c

```diff
@@ -2301,11 +2313,19 @@ static int test_is_repo(url_t *url)
         buf2,
         URL_FLAG_PROGRESS + URL_FLAG_UNZIP + opt * URL_FLAG_OPTIONAL
       )) {
-        log_info("mount %s -> %s\n", file_name, sl->value);
-
-        i = util_mount_ro(file_name, sl->value, url->file_list) ? 0 : 1;
-        ok &= i;
-        if(!i) log_info("instsys mount failed: %s\n", sl->value);
+        if(copy) {
+          char *dst = strrchr(t, '/') ?: t;
```

Comment:
Arvin just found out that this is a GNU extension. But that's good enough for us since we can rely on having GNU C.